css/login-page.css

```diff
@@ -0,0 +1,69 @@
+body{
+    background-color: gray;
+    background-image: url("background-soft4.jpeg");
+    background-repeat: no-repeat;
+    overflow: hidden;
+}
+
+div#forma{
+    width: 300px;
+    margin: 0 auto;
+    padding-top: 40px;
+    display: block;
+    text-align: center;
+}
+
+h1{
+    text-align: center;
+    color: white;
+}
+
+div.jumbotron-fluid{
+    background-color: darkslategray;
+    width: 600px;
+    margin: 80px auto;
+}
+
+img#img-jbt{
```

Comment:
Every time you need to name something try to find a name already states what that portion of code does without reading it.
Especially for functions